mesonbuild/compilers/compilers.py

```diff
@@ -0,0 +1,982 @@
+# Copyright 2012-2014 The Meson development team
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shutil
+import contextlib
+import subprocess, os.path
+import tempfile
+from .. import mesonlib
+from ..linkers import *
```

Comment:
__[flake8]__
* [F403] 'from ..linkers import *' used; unable to detect undefined names

<!-- Comment by [SideCI](https://sideci.com) -->